playbooks/tasks/portal-health-check-disable.yml

```
@@ -13,12 +12,12 @@
   register: result
 
 - name: Stop portal healh check to take portal out of load balancer
-  command: docker exec health-check cli/disable
+  command: docker exec health-check cli/disable {{ out_of_lb_message }}
```

```suggestion
  command: "docker exec health-check cli/disable {{ out_of_lb_message }}"
```

rows with vars in templates must be quoted